src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Person } from '../types';
+import { findPersonByName } from '../utils/findPersonByName';
+
+interface Props {
+  person: Person;
+  people: Person[];
+}
+
+export const PersonInfo: FC<Props> = ({ person, people }) => {
+  const { personSlug } = useParams();
+
+  const mother = findPersonByName(people, person.motherName);
+  const father = findPersonByName(people, person.fatherName);
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': person.slug === personSlug })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={cn({
+            'has-text-danger': person.sex === 'f',
+          })}
+        >
+          {person.name}
+        </Link>
```

Comment:
You are using a string template for the link here. Consider using a function or a constant to build URLs. This will make your code more maintainable because you will not have to change the URL in many places if it changes in the future.